replication/event.go

```diff
@@ -299,6 +299,85 @@ func (e *GTIDEvent) Dump(w io.Writer) {
 	fmt.Fprintln(w)
 }
 
+type BeginLoadQueryEvent struct {
+	FileID    uint32
+	BlockData []byte
+}
+
+func (e *BeginLoadQueryEvent) Decode(data []byte) error {
+	pos := 0
+
+	e.FileID = binary.LittleEndian.Uint32(data[pos:])
+	pos += 4
+
+	e.BlockData = data[pos:]
+
+    return nil
+}
+
+func (e *BeginLoadQueryEvent) Dump(w io.Writer) {
+	fmt.Fprintf(w, "File ID: %d\n", e.FileID)
+	fmt.Fprintf(w, "Block data: %s\n", e.BlockData)
```

Comment:
Yes, the BlockData is human readable.

According to [BEGIN_LOAD_QUERY_EVENT](https://dev.mysql.com/doc/internals/en/begin-load-query-event.html), block-data is string.EOF, just like [QUERY_EVENT](https://dev.mysql.com/doc/internals/en/query-event.html) query.